pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhc-pig";
```

```suggestion
  pname = "dhcpig";
```
I don't think that using a dash in the middle when upstream doesn't is necessary.